Python/ceval_macros.h

```diff
@@ -144,7 +144,8 @@ GETITEM(PyObject *v, Py_ssize_t i) {
 #define NEXTOPARG()  do { \
         _Py_CODEUNIT word = *next_instr; \
         opcode = _Py_OPCODE(word); \
-        oparg1 = oparg = _Py_OPARG(word); \
+        oparg = _Py_OPARG(word); \
+        oparg1 = oparg; \
```

Comment:
I'm not excited about having two variables. In my own 'long-insts' branch I just count `oparg`, `oparg2`, `oparg3`, ...

Reply:
I think the generator currently emits REG(oparg1). We could change that.

Reply:
Or we could "#define oparg1 oparg".

Reply:
I prefer modifying the generator over the macro. REG(oparg1) only occurs once in the generator, explicitly, so it's very easy to change (in fact in ly 'long-instrs' I already changed it.)

Reply:
You can even cherry-pick 8d06d1b1e5 from that branch.